tartiflette_plugin_scalars/naive_datetime.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime
+from typing import Union  # pylint: disable=unused-import
```

Comment:
Sorry i didn't catch it before, but `Union` seems to be used :) you can remove the "disable"

Reply:
Ok will do! Also did you mean next.md as in 0.2.0.md?

Reply:
No, no, next.md is perfect, i'll do the release PR after.